src/locale/messages.da.xlf

```diff
@@ -465,9 +461,13 @@
       </trans-unit>
       <trans-unit id="2994423491366084246" datatype="html">
         <source>dialog.message.rehearse.warning</source>
-        <target state="new">Er du sikker på at du vil afprøve denne Rundown?
+        <target state="new">Er du sikker på at du vil afprøve denne Rundown? 
         Det vil deaktivere Rundown</target>
       </trans-unit>
+      <trans-unit id="3477258835430959446" datatype="html">
+        <source>keyboard-mapping-settings-page.delete-all-keyboard-mappings.success <x id="PH" equiv-text="actionTriggerIdsToBeDeleted.length"/></source>
+        <target state="new"><x id="PH" equiv-text="actionTriggerIdsToBeDeleted.length"/> genvejene blev slettet </target>
```

Comment:
The danish translation here would be `{{yourVariableHere}} genveje blev slettet` :)

Reply:
Fixed